src/test/java/wooteco/subway/line/dao/JDBCLineDaoTest.java

```diff
@@ -0,0 +1,111 @@
+package wooteco.subway.line.dao;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.data.jdbc.DataJdbcTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.TestConstructor;
+import wooteco.subway.exception.NotFoundLineException;
+import wooteco.subway.line.domain.Line;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJdbcTest
+@TestConstructor(autowireMode = TestConstructor.AutowireMode.ALL)
+class JDBCLineDaoTest {
+
+    private final JDBCLineDao jdbcLineDao;
+
+    public JDBCLineDaoTest(JdbcTemplate jdbcTemplate) {
+        this.jdbcLineDao = new JDBCLineDao(jdbcTemplate);
+    }
+
+    @Test
+    @DisplayName("노선 추가 테스트")
+    void save() {
+        Line line = new Line("2호선", "bg-red-600");
+
+        Line savedLine = jdbcLineDao.save(line);
+
+        assertThat(savedLine.getName()).isEqualTo(line.getName());
+        assertThat(savedLine.getColor()).isEqualTo(line.getColor());
+    }
+
+    @Test
+    @DisplayName("노선 조회 테스트")
+    void findAll() {
+        Line line1 = new Line("2호선", "bg-red-600");
+        Line line2 = new Line("3호선", "bg-red-600");
+
+        Line savedLine1 = jdbcLineDao.save(line1);
+        Line savedLine2 = jdbcLineDao.save(line2);
+
+        List<Line> lines = jdbcLineDao.findAll();
+
+        assertThat(lines).hasSize(2);
+        assertThat(lines).containsExactly(savedLine1, savedLine2);
```

Comment:
```suggestion
        assertThat(lines).hasSize(2)
            .containsExactly(savedLine1, savedLine2);
```

https://github.com/woowacourse/atdd-subway-map/pull/85#discussion_r628778079 에서의 핵심은 체이닝이었는데 혹시 놓치신 걸까요?

Reply:
아하... 체이닝이였군요..! 놓쳤네요... 처음알았습니다! 감사합니다~